hazelcast/src/main/java/com/hazelcast/internal/nio/tcp/TcpIpUnifiedEndpointManager.java

```diff
@@ -25,18 +25,13 @@
 import com.hazelcast.logging.LoggingService;
 import com.hazelcast.spi.properties.HazelcastProperties;
 
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
-
 import static com.hazelcast.internal.metrics.MetricDescriptorConstants.TCP_METRIC_UNIFIED_ENDPOINT_MANAGER_CLIENT_COUNT;
 import static com.hazelcast.internal.metrics.MetricDescriptorConstants.TCP_METRIC_UNIFIED_ENDPOINT_MANAGER_TEXT_COUNT;
 import static com.hazelcast.internal.metrics.ProbeLevel.MANDATORY;
 import static com.hazelcast.internal.nio.ConnectionType.MEMCACHE_CLIENT;
 import static com.hazelcast.internal.nio.ConnectionType.REST_CLIENT;
 
-class TcpIpUnifiedEndpointManager
-        extends TcpIpEndpointManager {
+class TcpIpUnifiedEndpointManager extends TcpIpEndpointManager {
```

Comment:
Probably we can get rid of this class as well. The only thing it adds are some metrics; these could be add conditionally in the TcpIpEndpointManager,